### ISSUE TITLE
iceprod2 - globus proxies (Trac #848)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#842]
Handling of globus certificates is always fun.

Site proxy:
* Use a site proxy provided by the site admin as a general default.
  * Store location of certificate and passphrase in local config.
  * Regenerate usable proxy as needed.

User proxy: moved to #1444

Migrated from https://code.icecube.wisc.edu/ticket/848
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:28", 
    "description": "Handling of globus certificates is always fun.\n\nSite proxy:\n* Use a site proxy provided by the site admin as a general default.\n  * Store location of certificate and passphrase in local config.\n  * Regenerate usable proxy as needed.\n\nUser proxy: moved to #1452", 
    "reporter": "david.schultz", 
    "cc": "ddelventhal", 
    "resolution": "fixed", 
    "_ts": "1550067088921308", 
    "component": "iceprod", 
    "summary": "iceprod2 - globus proxies", 
    "priority": "major", 
    "keywords": "", 
    "time": "2015-01-06T20:36:40", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150504T17:04:26: 
* david.schultz changed milestone from "" to "IceProd 2.0"
* david.schultz commented:

> Site proxy for 2.0 milestone.  User proxy for 2.1 milestone.  Consider breaking this into two tickets.

**Comment by icecube-trac** [comment]
Trac update at 20150831T19:06:30: david.schultz changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20151125T20:10:04: david.schultz changed description from:

> Handling of globus certificates is always fun.
> 
> Site proxy:
> * Use a site proxy provided by the site admin as a general default.
>   * Store location of certificate and passphrase in local config.
>   * Regenerate usable proxy as needed.
> 
> User proxy (depends on #847):
> * Store a proxy under that user's account in the DB.
>   * A moderately long-lived proxy with ability to generate sub-proxies. Probably need some sort of script to make this for the user, from a certificate.
>   * Make limited sub-proxy to send with tasks by this user.
>   * Notify (email?) a user if proxy will expire soon and tasks are still running.

to:

> Handling of globus certificates is always fun.
> 
> Site proxy:
> * Use a site proxy provided by the site admin as a general default.
>   * Store location of certificate and passphrase in local config.
>   * Regenerate usable proxy as needed.
> 
> User proxy: moved to #1452

**Comment by icecube-trac** [comment]
Trac update at 20151125T22:28:19: 
* david.schultz commented:

> In changeset 139858:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="139858"
> add a basic site proxy interface. it's not ideal, but it will work until iceprod 2.1. closes #842
> ```


* david.schultz changed resolution from "" to "fixed"
* david.schultz changed status from "accepted" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:28: 
* olivas commented:

> Milestone IceProd 2.0 deleted


* olivas changed milestone from "IceProd 2.0" to ""
